task.js

```diff
@@ -0,0 +1,64 @@
+window.onload = function() {
+
+  const tasks = [];
+  const taskBtn = document.getElementById('task_button');
+
+  function createStatus(newRow, status){
+    const newCell = newRow.insertCell();
+    const newText = document.createTextNode(status);
+    newCell.appendChild(newText);
+  }
+
+  function deleteBtn(newRow, status, id){
+    const newCell = newRow.insertCell();
+    const newText = document.createElement('button');
+    newText.textContent = status;
+    newCell.appendChild(newText);
+    newText.addEventListener('click', () =>{
+      const taskTable = document.getElementById('targetTable');
+      let rowNum = taskTable.rows.length; 
+      while(rowNum-1>0){
+        if (rowNum>1) {
+          taskTable.deleteRow(rowNum-1);
+        }
+        rowNum -= 1
+      }
+      tasks.splice(id, 1)
+      tasks.forEach(function(task) {
```

Comment:
アロー関数を使った方が良いかと思いました。

```suggestion
      tasks.forEach((task) => {
```

Reply:
アロー関数にしました。